scout/server/blueprints/variant/templates/variant/components.html

```diff
@@ -161,12 +161,8 @@
   {% if case.chanjo2_coverage %}
     <div class="d-flex flex-wrap ms-1" data-bs-toggle="tooltip" title="Chanjo2 coverage reports">
       {% for gene in variant.genes %}
-        {{ chanjo2_report_form(institute, case, gene.hgnc_symbol, 'overview', gene.hgnc_id, "Gene coverage " + (gene.common.hgnc_symbol if gene.common else gene.hgnc_id), "btn btn-sm btn-secondary text-white") }} <!--chanjo2 genes overview -->
-        {% if gene.common.hgnc_id in gene_has_full_coverage and gene.common.hgnc_id[gene.common.hgnc_id] %}
-          <span class="bg-success fa-solid fa-circle-check" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Chanjo2 coverage is at 100% completeness."></span>
-        {% else %}
-          <span class="bg-warning fa-solid fa-triangle-exclamation" data-bs-toggle="tooltip" data-bs-placement="bottom" title="Note that Chanjo2 coverage is below 100% completeness."></span>
-        {% endif %}
+        {{ chanjo2_report_form(institute, case, gene.hgnc_symbol if gene.hgnc_symbol else gene.hgnc_id, 'overview', gene.hgnc_id, "Gene coverage " ~ (gene.hgnc_symbol if gene.hgnc_symbol else gene.hgnc_id), "btn btn-sm btn-secondary text-white") }} <!--chanjo2 genes overview -->
+        <span class="button complete-coverage-indicator btn-info text-white btn-sm" style="display:flex" id="coverage-indicator-{{gene.hgnc_id}}-button" data-bs-toggle="tooltip" data-bs-placement="bottom" data-loading-text="Checking completeness..." title="Checking completeness..."><span id="coverage-indicator-{{gene.hgnc_id}}-icon" class='blink_me fa-solid fa-circle-question'></span><span id="coverage-indicator-{{gene.hgnc_id}}-text">Checking</span>&nbsp;{{gene.hgnc_symbol}}</span>
```

Comment:
This works very nicely. About the looks of this button, but I noticed that sometimes it's aligned with the coverage button and sometimes it isn't:

<img width="583" alt="image" src="https://github.com/user-attachments/assets/46d3c9ad-11e3-4d5d-a2ae-6f7b4bc6d8e9" />

<img width="664" alt="image" src="https://github.com/user-attachments/assets/de745800-c0bd-4512-a77a-a635d575944b" />


Turning these 2 buttons into a button group could help here. Is it doable? If this is doable then you can also omit the name of the gene in the second button

Also the icon inside the button is misaligned with the text:

<img width="189" alt="image" src="https://github.com/user-attachments/assets/cb921656-c654-42f4-8d1b-0240dcf0dd3d" />

Reply:
Sure, lets try at least! It would be less repetitive, but slightly wider by default without the stacking.